src/Effect/Timer.js

```diff
@@ -1,27 +1,27 @@
 /* no-redeclare global exports */
-export function setTimeout(ms) {
+export function setTimeout_(ms) {
   return function (fn) {
     return function () {
       return setTimeout(fn, ms);
     };
   };
 }
 
-export function clearTimeout(id) {
+export function clearTimeout_(id) {
   return function () {
     clearTimeout(id);
   };
 }
 
-export function setInterval(ms) {
+export function setInterval_(ms) {
   return function (fn) {
     return function () {
       return setInterval(fn, ms);
     };
   };
 }
 
-export function clearInterval(id) {
+export function clearInterval_(id) {
```

Comment:
If we do this, then we should use `clearIntervalImpl` rather than the underscore (since that has a particular meaning in PureScript-land).